.github/workflows/validatePR.yml

```diff
@@ -0,0 +1,28 @@
+name: Validate PR
+
+on:  
+  pull_request:
```

Comment:
Would you want to run it only when a PR is generated or even when someone makes subsequent pushes?

Reply:
It still runs even on subsequent pushes against an opened PR. Ideally the `dev` branch should be closed off from direct pushes.

Reply:
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request

Reply:
Nice! I did not realize subsequent pushes were covered... 

> It still runs even on subsequent pushes against an opened PR. Ideally the `dev` branch should be closed off from direct pushes.